### PR TITLE
Expose MulticastDelegate.GetObjectData method

### PR DESCRIPTION
**Pull request by rahku**

### CODE REVIEW

**Comment by rahku** [comment]
This method is only required for serialization and is already being tested in serialization tests. So not adding any new test.

**Comment by rahku** [comment]
PTAL @weshaggard

File: src/mscorlib/model.xml [hunk 1]
```diff
@@ -3413,6 +3413,7 @@
       <Member Name="GetHashCode" />
       <Member Name="GetInvocationList" />
       <Member Name="GetMethodImpl" />
+      <Member Name="GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
```
Comment (stephentoub) [inline src/mscorlib/model.xml]: I'm just curious.  It's fine to be explicit about it, but this was already getting output in the DLL due to it being the implementation of an interface method, right?

Comment (rahku) [inline src/mscorlib/model.xml]: Yes correct Implementation was already present.

**Comment by weshaggard** [comment]
LGTM

@rahku I think it would be better to do a larger pass over S.P.CoreLib and expose all the APIs you plan to expose as that will help cut down on the round-tripping between coreclr and corefx.

**Pull request merged**